example_configs/tomcat.yml

```diff
@@ -1,33 +1,680 @@
 ---   
+# Example jmx_exporter configuration for use with Tomcat 8.
+
 lowercaseOutputLabelNames: true
 lowercaseOutputName: true
-rules:
-- pattern: 'Catalina<type=GlobalRequestProcessor, name=\"(\w+-\w+)-(\d+)\"><>(\w+):'
-  name: tomcat_$3_total
+
+whitelistObjectNames: ["Tomcat:*", "java.lang:*"]
+
+# ------------------------ GlobalRequestProcessor -----------------------
+
+# maxTime
+- pattern: '^Tomcat<type=GlobalRequestProcessor, name=\"(.*)\"><>maxTime:'
+  name: tomcat_global_processingtime_milliseconds_max
+  help: 'Tomcat global maximum request processing time seen, in milliseconds.'
```

Comment:
Thanks for the reminder. I'll do the multiplication.

Do you mean the max is not very useful?